tracing/span.go

```diff
@@ -33,7 +40,7 @@ const (
 type Span struct {
 	tracer *Tracer
 
-	name     string
+	Name     string
```

Comment:
style nit: move the exported fields to the top, before all the unexported fields.